File: StokryptoIOS/Stokrypto/Data/Models/ServiceModel.swift
//
//  ServiceModel.swift
//  Stokrypto
//
//  Created by bach.bui on 14/07/2023.
//

import Foundation

class ServiceModel: Service {
    var id: String
    var name: String
    var icon: String
    var title: String
    var subtitle: String
    var refId: String
    
    init(id: String, name: String, icon: String, title: String, subtitle: String = "", refId: String) {
        self.id = id
        self.name = name
        self.icon = icon
        self.title = title
        self.subtitle = subtitle
        self.refId = refId
    }
}

File: StokryptoAndroid/app/src/main/java/app/web/bachbntdev/stokrypto/data/models/ServiceModel.kt
package app.web.bachbntdev.stokrypto.data.models

import app.web.bachbntdev.stokrypto.domain.entities.Service

data class ServiceModel(
    override val id: String,
    override val name: String,
    override val icon: Int,
    override val title: String,
    override val subtitle: String = "",
    override val refId: String
) : Service